pytest.ini

```diff
@@ -4,12 +4,11 @@ xvfb_height = 1080
 # use this due to some issues with ndarray reshape errors on CI systems
 xvfb_colordepth = 24
 xvfb_args=-ac +extension GLX +render
-addopts = --faulthandler-timeout=15
```

Comment:
Place whatever the new `pytest.ini` entry pytest 5.0+ is looking for and I think we're good.